instana-java-sdk-sample/src/main/java/com/acme/batch/CustomBatchJob.java

```diff
@@ -12,11 +12,12 @@
    * This instrumentation will create traces for each invocation of this "batch job". The Instana default
    * instrumentation will not see it, as no standard framework, like Spring Batch, is used.
    */
-  @Span(value = "my-custom-batch", captureArguments = true, captureReturn = true)
+  @Span(value = "my-custom-batch", captureArguments = true, captureReturn = true, type = Span.Type.ENTRY)
```

Comment:
a very small detail, it seems that we usually put `type` as the first argument of the `@Span` annotation
e.g. https://github.com/instana/instana-java-sdk/blob/master/instana-java-sdk-sample/src/main/java/com/acme/tcp/server/CustomTCPClient.java#L14